dataCollection/settings.py

```diff
@@ -8,7 +8,7 @@
 #TEMPLATE_DEBUG = DEBUG
 
 ADMINS = (
-    ('Tom Moore', 'tjm3772@rit.edu'),
+    ('Sally PandaFace', 'spf3000@rit.edu'),
```

Comment:
Nope, I changed the data to not have real stuff. Like 'USERNAME' and 'PASSWORD' ;)